manifests/init.pp

```diff
@@ -938,14 +938,26 @@
       case $::osfamily {
         'RedHat': {
 
-          file { 'pam_system_auth_ac':
-            ensure  => file,
-            path    => $system_auth_ac_file,
-            content => template('pam/system-auth-ac.erb'),
-            owner   => 'root',
-            group   => 'root',
-            mode    => '0644',
-            require => Package[$my_package_name],
+          if $ensure_vas == 'present' {
```

Comment:
inside the if statement use the reference syntax so the code is not duplicated such as the following.

``` puppet
File['pam_system_auth_ac'] {
  require => [ Package[$my_package_name], Exec[vasinst], ],
}
```